src/App.tsx

```diff
@@ -17,18 +48,34 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                query={query}
+                sFilter={setFilter}
+                sQuery={setQuery}
+              />
             </div>
 
             <div className="block">
-              <Loader />
-              <TodoList />
+              {!todos.length ? (
```

Comment:
it's better to create an additional state `isLoading` for it. What if in the server no todos (and we get an empty array in response) - we will show infinity loader